mev_inspect/compound_liquidations.py

```diff
@@ -54,7 +80,18 @@ def get_compound_liquidations(
                 trace.transaction_hash, trace.trace_address, traces
             )
             seize_trace = _get_seize_call(child_traces)
-            if seize_trace is not None and seize_trace.inputs is not None:
+            if trace.protocol == Protocol.compound_v2:
+                underlying_markets = collateral_by_c_token_address
+            elif trace.protocol == Protocol.cream:
+                underlying_markets = collateral_by_cr_token_address
+            else:
+                underlying_markets = {}
+
+            if (
+                seize_trace is not None
+                and seize_trace.inputs is not None
+                and underlying_markets is not {}
```

Comment:
`is` compares object identity and should only be used for `None` checks
```
>>> {} is {}
False
```

should be `len(underlying_markets) == 0`